docs/tutorials/tutorial-kafka.md

```diff
@@ -24,260 +24,267 @@ sidebar_label: "Load from Apache Kafka"
+   Druid's architecture requires that you specify a primary timestamp column. Druid stores the timestamp in the `__time`) column in your Druid datasource.
+   In a production environment, if you don't have a timestamp in your data, you can select **Parse timestamp from:** `none` to use a placeholder value. 
 
-Once you are satisfied with the schema, click `Next` to go to the `Partition` step where you can fine tune how the data will be partitioned into segments.
+   For the sample data, the data loader selects the `timestamp` column in the raw data as the primary time column.
 
-![Data loader partition](../assets/tutorial-kafka-data-loader-06.png "Data loader partition")
+6. Click **Next: ...** three times to go past the **Transform** and **Filter** steps to **Configure schema**. You don't need to enter anything in these two steps because applying transforms and filters is out of scope for this tutorial.
 
-Here, you can adjust how the data will be split up into segments in Druid.
-Since this is a small dataset, there are no adjustments that need to be made in this step.
+   ![Data loader schema](../assets/tutorial-kafka-data-loader-05.png "Data loader schema")
 
-Click `Next: Tune` to go to the tuning step.
+7. In the **Configure schema** step, you can select data types for the columns and configure [dimensions](../ingestion/data-model.md#dimensions) and [metrics](../ingestion/data-model.md#metrics) to ingest into Druid. The sample data contains three nested columns, so you need to create JSON-type dimensions for them. 
 
-![Data loader tune](../assets/tutorial-kafka-data-loader-07.png "Data loader tune")
+    Click **Add dimension** and enter the following information. You can only add one dimension at a time.
+    - Name: `event`, Type: `json`
+    - Name: `agent`, Type: `json`
+    - Name: `geo_ip`, Type: `json`
+  
+    After you create the dimensions, you can scroll to the right in the preview window to see the nested columns:
 
-In the `Tune` step is it *very important* to set `Use earliest offset` to `True` since we want to consume the data from the start of the stream.
-There are no other changes that need to be made here, so click `Next: Publish` to go to the `Publish` step.
+    ![Nested columns schema](../assets/tutorial-kafka-data-loader-05b.png "Nested columns schema")
 
-![Data loader publish](../assets/tutorial-kafka-data-loader-08.png "Data loader publish")
+8.  Click **Next: Partition** to configure how Druid partitions the data into segments.
 
-Let's name this datasource `wikipedia-kafka`.
+    ![Data loader partition](../assets/tutorial-kafka-data-loader-06.png "Data loader partition")
 
-Finally, click `Next` to review your spec.
+9.  Select `day` as the **Segment granularity**. Since this is a small dataset, you don't need to make any further adjustments. Click **Next: Tune** to finetune how Druid ingests data.
```

Comment:
```suggestion
9.  Select `day` as the **Segment granularity**. Since this is a small dataset, you don't need to make any further adjustments. Click **Next: Tune** to fine-tune how Druid ingests data.
```
Raised by Travis

Reply:
Google style guide says to avoid hyphenating words unless it's essential for their meaning.